src/screens/accountCreationScreen.js

```diff
@@ -20,15 +40,28 @@ const AccountCreationScreen = () => {
           </View>
           <TouchableWithoutFeedback>
             <View style={styles.profilePhoto}>
-              <Text style={styles.buttonText}>Select a profile photo</Text>
+              <Text style={styles.profilePhotoText}>
+                Select a profile photo
+              </Text>
             </View>
           </TouchableWithoutFeedback>
           <View style={styles.verfificationBox}>
-            <Text style={styles.title}>Name</Text>
-            <TextInput />
-            <Text style={styles.title}>Phone number</Text>
-            <TextInput />
+            <Text style={styles.infoTitleText}>Name</Text>
+            <TextInput
+              style={styles.textInput}
+              value={userInfo.userName}
+              onChangeText={(e) => changeUserInfo({displayName: e})}
```

Comment:
See above regarding set state. If ```userInfo``` has other fields, ```changeUserInfo({displayName: e})}``` means ```userInfo === {displayName: e}```. 
```suggestion
              onChangeText={(e) => setUserInfo({... userInfo, displayName: e})}
```

The above suggestion will only set the displayName field while keeping other fields the same.